akka-cluster/src/main/scala/akka/cluster/VectorClock.scala

```diff
@@ -187,5 +187,11 @@ final case class VectorClock(
     VectorClock(mergedVersions)
   }
 
+  def prune(removedNode: Node): VectorClock =
```

Comment:
depends on you mental model of the VectorClock. I normally see it as a table:

    node    version
    node-1: 3
    node-2: 1
    node-3: 6

`prune(node-3)` would result in:

    node    version
    node-1: 3
    node-2: 1

i.e. removing a row in the table